lib/schema/operations.graphql

```diff
@@ -25,7 +25,39 @@ query FindUsers ($filter: UserListFilter) {
     }
 }
 
+query FindUsersWithFilter ($filter: UserListFilter) {
```

Comment:
Why do we have this separately to `FindAllUsers`? Isn't it better to add `jobTitle` as a field in `FindUsers`?

Reply:
These two queries have different purposes, even though they call the same endpoint. We do not need the jobTitle field for all users (not yet at least), so we instead query a list of users by ID using the filter function defined here. We could combine several queries into one query with all of the fields we could ever need, but then we lose the benefits of using GraphQL which is that we query exactly what we need when we need it.